inference-engine/tests/functional/plugin/cpu/shared_tests_instances/subgraph_tests/reshape_squeeze_reshape_relu.cpp

```diff
@@ -31,19 +31,16 @@ namespace {
                                                              InferenceEngine::Precision::FP16,
     };
 
-    INSTANTIATE_TEST_CASE_P(reshape_squeeze_reshape_relu, ReshapeSqueezeReshapeRelu,
-                            ::testing::Combine(
-                                    ::testing::ValuesIn(inputs),
-                                    ::testing::ValuesIn(netPrecisions),
-                                    ::testing::Values(CommonTestUtils::DEVICE_CPU),
-                                    ::testing::Values(true)),
-                            ReshapeSqueezeReshapeRelu::getTestCaseName);
+    const std::vector<ngraph::helpers::SqueezeOpType> opTypes = {
+            ngraph::helpers::SqueezeOpType::SQUEEZE,
+            ngraph::helpers::SqueezeOpType::UNSQUEEZE
```

Comment:
Why did you add additional unsqueeze case for certain subgraph which intended to be exactly reshape->squeeze->reshape?

Reply:
It's not additional test case, It was handles in test definition by flags.